MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -421,9 +425,9 @@ open class RouteLegProgress: NSObject {
      */
     @available(*, deprecated, message: "Use RouteProgress.nearbyCoordinates")
     public var nearbyCoordinates: [CLLocationCoordinate2D] {
```

Comment:
This property has long been deprecated. Can we delete this property and the coordinate array concatenation along with it?